ddtrace/llmobs/_writer.py

```diff
@@ -141,6 +142,7 @@ def __init__(
         _api_key: str = "",
         _app_key: str = "",
         _override_url: str = "",
+        _default_project: Project = Project(name="", _id=""),
```

Comment:
this default is duplicated between here and the `_llmobs.py` usage. Probably better to just opt for one of them (likely just this one)